pkg/initialdata/initialdata.go

```diff
@@ -140,15 +139,23 @@ func CheckDatastore(ctx context.Context, ds datastore) (bool, error) {
 }
 
 // InsertIntoDatastore inserts the given JSON data into datastore with write requests.
+//
+// The given data should be formatted like this: '{"collection_a": {"1": {"field_a": "foo", "field_b": "bar"}}}'
+// but the datastore.Create expected a map with FQId like this: '{"collection_a/1": {"field_a": "foo", "field_b": "bar"}}'
+// so this function also transforms the data into correct format.
 func InsertIntoDatastore(ctx context.Context, ds datastore, data []byte) error {
 	var d map[string]json.RawMessage
```

Comment:
The variable d is used for 25 lines. You should gibe it a better name

Reply:
Done.